source/crud/bulk-write.md

```diff
@@ -622,6 +627,11 @@ server. Drivers MUST explicitly define `ordered` as `true` in the `bulkWrite` co
 `BulkWriteOptions`. This is required to avoid inconsistencies between server and driver behavior if the server default
 changes in the future.
 
+Drivers MUST return a client-side error if `ordered` is true (including when default is applied) with an unacknowledged
```

Comment:
What should drivers that already released this api do? Should we continue supporting it until the next major version? Drop it in a minor version? We usually try to never do the latter but I'd be willing to make an expection here since unack'd writes are so niche and the fix is simple, just add ordered=False.

Reply:
Just to clarify, Python does not have that bug. We send w:1 so we can stop if a batch fails.

Reply:
@ShaneHarvey: what is the bug you're referring to? I'm a bit confused how PyMongo sending `w:1` relates to the user specifying `w:0` with `ordered:true`. Does that mean PyMongo is already disregarding the user's original write concern in order to catch failed batches?